packages/next/src/views/List/resolveAllFilterOptions.ts

```diff
@@ -28,8 +30,27 @@ export const resolveAllFilterOptions = async ({
           siblingData: {}, // use empty object to prevent breaking queries when accessing properties of data
           user: req.user,
         })
+
         resolvedFilterOptions.set(field.name, options)
       }
+
+      if (fieldHasSubFields(field)) {
+        await resolveAllFilterOptions({
+          fields: field.fields,
+          req,
+          result: resolvedFilterOptions,
+        })
+      }
+
+      if (field.type === 'tabs') {
+        field.tabs.forEach(async (tab) =>
+          resolveAllFilterOptions({
```

Comment:
either missing `await`, or `void` prefix if intentional - in that case no need for the `async` keyword

Reply:
Good catch